src/CharacterList.js

```diff
@@ -53,7 +53,7 @@ class CharacterList extends Component {
       <div>
         <h2 className="main_title">Characters</h2>
         <ul className="nav_items">
-          {alphabet.map((letter, index) => <AlphaLinks letter={letter} key={index} />)}
+          {alphabet.map((letter, index) => <AlphaLinks letter={letter} key={index} selected={this.state.letter} />)}
```

Comment:
I guess I was thinking that the component with the class would make the logical determination of equality, rather than relying on the parent to do that. I think the name here of "selected" infers a boolean, though, so I can see where it would be confusing.